codeStyle.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component name="ProjectCodeStyleSettingsManager">
+  <option name="PER_PROJECT_SETTINGS">
+    <value>
+      <option name="RIGHT_MARGIN" value="120"/>
+      <option name="INDENT_SIZE" value="4"/>
+      <option name="CONTINUATION_INDENT_SIZE" value="4"/>
+      <option name="TAB_SIZE" value="4"/>
+      <option name="USE_TAB_CHARACTER" value="false"/>
+      <codeStyleSettings language="JAVA">
+        <option name="KEEP_LINE_BREAKS" value="true"/>
```

Comment:
please check what changes, when you do a formatting of existing code. it should not be too invasive. do you plan to follow up with a reformatting PR?